### PR TITLE
Changes made since 'modserv' fork

### PR DESCRIPTION
**Pull request by wmhilton**
Looks like the major bullet points are:

1. [ ] support separate subdomains for the registry API (e.g. https://registry.node-modules.io) and the UI (e.g. https://ui.node-modules.io)
2. [ ] support for a JSON config file
3. [x] updated the old skimdb URL to the new one that includes scoped modules
4. [ ] aaaaand a Dockerfile! 

Which feature do you want? I'm gonna suggest we make separate PR issues for them and do one feature at a time.

### CODE REVIEW

File: lib/www/views/main.html [hunk 1]
```diff
@@ -2,12 +2,12 @@
 <html ng-controller="MainCtrl">
   <body>
     <div class="jumbotron">
-      <h1 class="sr-only">local npm</h1>
+      <h1 class="sr-only">modserv</h1>
```
Comment (gabrielcsapo) [inline lib/www/views/main.html]: can we keep the local npm naming?

File: lib/www/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>local-npm</title>
```
Comment (gabrielcsapo) [inline lib/www/index.html]: revert naming

**Comment by gabrielcsapo** [comment]
This is awesome! I think all 4 options are valid and should be added back to the main project! Really awesome stuff, very excited!

**Comment by gabrielcsapo** [comment]
I am going to submit a cleanup PR tomorrow or Monday and would appreciate your input. I am trying to condense and cleanup the project so we can add features more easily. What do you think @wmhilton?

**Comment by wmhilton** [comment]
Does that mean you're refactoring? It might be easier to add *these* features before you do that. Or at least the small ones.

I would like to make "3. updated the old skimdb URL to the new one that includes scoped modules" a priority, since it's more like a bug fix than a feature. I'll open a PR for it.

**Comment by wmhilton** [comment]
P.s. Can you add me to the local-npm organization so I can help clean up the repo? There's 40+ stale branches dating back years which makes it harder to use branches.

**Comment by gabrielcsapo** [comment]
I was thinking we add the features after we refactor? @wmhilton Adding you now

**Comment by gabrielcsapo** [comment]
@wmhilton just deleted those branches, wow a ton of branches 😂!

**Comment by wmhilton** [comment]
One down, three to go! :) I'll let you keep refactoring. But let's briefly talk about **config files**! They're awesome. Since you're refactoring, I would go a different route than I took in my original, which was a hackish `--config <file>` flag. My favorite config file library (which I didn't know about at the time) is [`rc`](https://www.npmjs.com/package/rc) by the evergreen genius @dominictarr. I love it because it takes zero effort on the developer's side, but handles every conceivable edge case for the user. It allows config options to be specified as command line arguments, environment variables, in .json or .ini files (in the working directory, the home directory, the system /etc directory, or the file specified via `--config`), and merges them all into one options object.

**Comment by gabrielcsapo** [comment]
@wmhilton the refactor effort right now is to update the tests to be more maintainable and cleaning up the directory. Code level changes I will leave to future discussion. I like the idea of adding rc! let's open an issue and anyone can pick up that effort!

**Comment by gabrielcsapo** [comment]
@wmhilton where are you 😢

**Comment by gabrielcsapo** [comment]
@wmhilton pinnnngg

**Pull request closed without merging**